README.md

```diff
@@ -71,9 +71,11 @@ bundle install --without development
 ## Usage
 
 ```shell
-./mp3lyrics.rb <dir> [override]
+./mp3lyrics.rb <dir> [-override true/false] [-use lyricwikia/genius/metrolyrics/azlyrics/swiftlyrics]
```

Comment:
one last issue: README states to use `-use` CLI argument, the code checks for `-wiki` - pick one you prefer 👍